frontend/src/pages/ExplorePage.tsx

```diff
@@ -202,21 +217,72 @@ const ExplorePage = () => {
   const [languageOpen, setLanguageOpen] = useState(false);
   const [openProfile, setOpenProfile] = useState(false);
   const [isFilterOpen, setIsFilterOpen] = useState(false);
+  const [favourite, setFavourite] = useState("");
   const [gender, setGender] = useState<string>("");
-  const [locations, setLocations] = useState<{ [key: string]: boolean }>({});
+  const [searchInput, setSearchInput] = useState("");
+
+  const toiletCollection = [
```

Comment:
Can you chuck these into a json file and import them so they can be used in other components?